Source/Particles/WarpXParticleContainer.H

```diff
@@ -185,7 +185,7 @@ public:
                         const amrex::MultiFab& Bz) = 0;
 
     void DepositCharge(amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
-                       bool local = false);
+                       bool local = false, bool reset = true);
```

Comment:
If multiple species deposit charge, the default behavior is that `rho` contains only the charge deposited by the last one. I am wondering of this could lead to some surprising behavior in the future, if someone uses this function for something else, since this is not the same behavior as other routines like `DepositCurrent`. Is there a reason for not setting the default value for `reset` to `false`?

Reply:
Yes, definitely.